packages/dashboard-core-plugins/src/panels/WidgetPanelTooltip.tsx

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 import { CopyButton } from '@deephaven/components';
 import { GLPropTypes, LayoutUtils } from '@deephaven/dashboard';
 import './WidgetPanelTooltip.scss';
-import { ReactElement } from 'react-markdown';
 import type { Container } from '@deephaven/golden-layout';
+import { ReactElement } from 'react-markdown/lib/react-markdown';
```

Comment:
I don't know why this was imported from `react-markdown` to begin with, but it should just be imported from react. Or just use `React.ReactElement` in place of this type

In general, importing from paths that aren't the package root should be avoided unless the package says to do it (they can set up import paths for different import specifiers) or there is no other way to do it (like the CSS probably)

When you import from just the package name, the package's `package.json` is consulted for what file the user should be given. When you append paths, `package.json` is ignored and the import is completely dependent on the file structure of the package. These imports are much more brittle to any sort of rename